app/helpers/filters_helper_overrides.rb

```diff
@@ -13,11 +13,11 @@ def self.search_path(class_or_engine_name)
     @@overrides[deconstantize_name(class_or_engine_name)].call(class_or_engine_name)
   end
 
-  private
-
   def self.deconstantize_name(name)
     name.include?('::') ? name.deconstantize : name
   end
 
+  private
```

Comment:
I moved the class method `deconstantize_name` out of the private section due to IneffectiveAccessModifier